### ISSUE TITLE
[1.12.x] Custom animations not working in main menu

### ISSUE THREAD

**Issue by sp614x** [sp614x/optifine#920]
Since 1.12 the custom animations in main menu do not work.
Old check was:
```
        if (!this.isGamePaused)
        {
            this.renderEngine.tick();
        }
```
in 1.12 the check is:
```
        if (this.world != null)
        {
            this.renderEngine.tick();
        }
```

**Comment by sp614x** [comment]
In 1.11.x all animations stopped when the game was paused.
In 1.12.x the animations continue when the game is paused.

**Comment by sp614x** [comment]
This seems to be a vanilla bug: https://bugs.mojang.com/browse/MC-118001
It could be fixed in 1.13.x.
Still have to find a workaround for for 1.12.x.

**Comment by sp614x** [comment]
Updated preview C6 for 1.12.2
- fixed custom texture animations not working in main menu (#920)
